packages/number-field/src/vaadin-number-field.js

```diff
@@ -187,13 +224,19 @@ export class NumberField extends InputFieldMixin(InputSlotMixin(ThemableMixin(El
     this._increaseValue();
   }
 
-  /** @protected */
+  /**
+   * @protected
+   * @override
+   */
   _createConstraintsObserver() {
     // NOTE: do not call "super" but instead override the method to add extra arguments
     this._createMethodObserver('_constraintsChanged(required, min, max, step)');
```

Comment:
Since we have the static `constraints` getter, I wonder if these constraints could be listed in that getter instead of overriding `_createConstraintsObserver`?

Reply:
Done.